lib/mayaUsd/nodes/layerManager.cpp

```diff
@@ -114,7 +114,16 @@ MStatus disconnectCompoundArrayPlug(MPlug arrayPlug)
     return dgmod.doIt();
 }
 
-MayaUsd::LayerManager* findNode()
+static bool isFromReference(const MFnReference* fromReference, const MFnDependencyNode& node)
```

Comment:
Re-reading the code, this function name and getMayaReferenceOrigin in ProxyShapeBase.cpp are not the best names.

Maybe this function should be named isNodeFromCorrectSceneFile or something like that.